src/components/SetNewPassword/SetNewPassword.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from "react";
+import { Formik, Form } from "formik";
+// import { toast } from "react-toastify";
+import { useTranslation } from "react-i18next";
+// import { useNavigate } from "react-router-dom";
+
+import { validation } from "./validate";
+import ISetNewPassword from "./SetNewPassword.interface";
+import { paths } from "config/paths";
+import { Navbar } from "components";
+import { Title, ButtonForm, Footer } from "./SetNewPassword.style";
+import { Input, IconPassword, IconText, LogoPage, IconEye, IconEyeHide, Line } from "styles";
+import {
+  LoginForm,
+  StyledInlineErrorMessageForm,
+  LabelStyle,
+  FooterWrapperLeft,
+  LinkFooter,
+  FooterWrapperRight,
+  View,
+} from "../Login/Login.style";
+
+export const SetNewPassword = () => {
```

Comment:
czemu to jest komponentem? czy ten element będzie wykorzystywany na wielu stronach?